src/constants/translations/en/admin-table.json

```diff
@@ -0,0 +1,11 @@
+{
+  "adminsTab": "Admins Tab",
+  "name": "Name",
+  "email": "Email",
+  "lastLogin": "Last login",
+  "signedUp": "Signed up",
+  "all": "All",
+  "status": "Status",
+  "active": "Active",
+  "blocked": "Blocked"
```

Comment:
Create the same for Ukrainian language.

Reply:
Admin portal won't have Ukrainian translations according to our design.